markdowns/ocean-py/README.md

```diff
@@ -3,8 +3,8 @@ title: README.md
 slug: README.md
 app: ocean.py
 module: README
-source: https://github.com/oceanprotocol/ocean.py/blob/main/README.md
-version: 0.5.30
+source: https://github.com/oceanprotocol/ocean.py/blob/issue497-bumpversion-to-v0.7.0/README.md
```

Comment:
@DMats This can lead to 404 errors if the ocean.py branch `issue497-bumpversion-to-v0.7.0` gets deleted and the user clicks on the `View source` link. Is this branch going to exist till the next doc update?
Usually, the source of docs is `main` due to the following reasons:
1. Avoid 404 errors.
2. User can read docs corresponding to production instance(assuming `main`==production).
Otherwise, changes look good to me.

Reply:
I was wondering about this. Thanks for the clarification and the review! I updated it to use the main branch instead.